public/app/plugins/datasource/loki/queryUtils.test.ts

```diff
@@ -201,6 +202,23 @@ describe('parseToArray', () => {
   });
 });
 
+describe('obfuscate', () => {
+  it('obfuscates on invalid query', () => {
+    expect(obfuscate('{job="grafana"')).toEqual('{@@@=@@@@@@@@@');
+  });
+  it('obfuscates on valid query', () => {
+    expect(
+      obfuscate('sum(sum_over_time({test="test"} |= `` | logfmt | __error__=`` | unwrap test | __error__=`` [10m]))')
+    ).toEqual('sum(sum_over_time({@@@@=@@@@@@} |= @@ | logfmt | @@@@@@@@@=@@ | unwrap @@@@ | @@@@@@@@@=@@ [10m]))');
```

Comment:
Why __error__?

Reply:
Do you mean why ```  __error__=`` ``` was replaced? Basically `__error__` is just another label added by Loki, thus we are parsing it as a `Identifier` and replacing it.

Reply:
Changed the code to support it, let me know what you think.